Common/include/CConfig.hpp

```diff
@@ -2106,22 +2139,133 @@ class CConfig {
    */
   bool GetSpecies_StrongBC() const { return Species_StrongBC; }
 
+  /*!
+   * \brief Get the flame offset (point on plane separating burnt and unburnt zone)
+            for flamelet model initialization.
+   * \return flame offset for flamelet model initialization
+   */
+  su2double *GetFlameOffset(void) { return flame_offset; }
+
+  /*!
+   * \brief Get the flame normal (of the plane separating burnt and unburnt zone)
+            for flamelet model initialization. The normal points in the direction of the burnt zone.
+   * \return flame offset for flamelet model initialization
+   */
+  su2double *GetFlameNormal(void) { return flame_normal; }
+
+  /*!
+   * \brief Get the flame thickness (reaction zone) for flamelet model initialization.
+            This is the thickness of the transition layer from unburnt to burnt conditions.
```

Comment:
Please put all the flamelet stuff in a struct like we did with turbulence.

Reply:
Good idea! I will make a struct for all the variables connected to initialization of the flame solution.

Reply:
@pcarruscag So should we merge all these config options into one option FLAMELET_INITIALIZATION that sets up the initial solution of the flame? We cannot use them independently anyway
So those are the config options FLAME_NORMAL, FLAME_OFFSET, FLAME_THICKNESS, FLAME_BURNT_THICKNESS. They determine a plane in 3D space, on one side we impose unburnt conditions, on the other side we impose burnt conditions (conditions obtained from the lookup table).
This will then become a list of 8 doubles. For anything more complicated, the python wrapper can be used.

Reply:
Sounds good if it's used all together